### PR TITLE
Docs: Add prerequisites to enable auth.jwt for url login

### PR DESCRIPTION
**Pull request by eleijonmarck**
**why/what**
We need to add doc for prerequirestes for enabling the auth.jwt before adding the url_login parameter for jwt authentication

notes
original issue - https://github.com/grafana/grafana/issues/57072

### CODE REVIEW

**Comment by eleijonmarck** [comment]
@Jguer should we include this prerequisites in some of the other ways you could enable jwt?

Like the https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/jwt/#verify-token-using-a-json-web-key-set-loaded-from-https-endpoint

Or is HEADER and auth.jwt enabled not needed there

File: docs/sources/setup-grafana/configure-security/configure-authentication/jwt/index.md [hunk 1]
```diff
@@ -82,6 +82,10 @@ can use URL login instead.
 > **Warning**: this can lead to JWTs being exposed in logs and possible session hijacking if the server is not
 > using HTTP over TLS.
 
+#### Prerequisites
+
+- [Enable JWT]({{< relref "#enable-jwt" >}}).
```
Comment (Jguer) [inline docs/sources/setup-grafana/configure-security/configure-authentication/jwt/index.md:87]: I think this might be better as a `> *Note*`

**Review by Jguer — APPROVED** [review]

File: docs/sources/setup-grafana/configure-security/configure-authentication/jwt/index.md [hunk 2]
```diff
@@ -82,6 +82,10 @@ can use URL login instead.
 > **Warning**: this can lead to JWTs being exposed in logs and possible session hijacking if the server is not
 > using HTTP over TLS.
 
+#### Prerequisites
```
Comment (eleijonmarck) [inline docs/sources/setup-grafana/configure-security/configure-authentication/jwt/index.md:85]:
```suggestion
> *Note*

You need to enable JWT with this setting as well
```

**Comment by marefr** [comment]
@eleijonmarck removed it from the changelog since we normally don't add docs changes to the changelog

**Pull request merged**